drafts/25-xxx-specifications.txt

```diff
@@ -498,11 +498,15 @@ Example syntax:
 
 pr02. The token-list may not be empty.
 
+pr10. The token-list shall not begin with the identifier `STDF`, either
+      before or after macro expansion.
```

Comment:
Should we also prevent the first token from beginning with `STDF`? I.e. reserve `#pragma STDF_ME` and the like?